### PR TITLE
Billing system

### PR DESCRIPTION
**Pull request by NothingIsMatter**
Added Billing Service which has method makeTransaction, added Wallet entity

### CODE REVIEW

File: backend/src/main/java/com/v2gdemo/backend/service/BillingService.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    Wallet wallet = walletRepository.findByAddress(address);
+    if (wallet == null) throw new Exception("Wrong address");
+    if (car.getWallet().getBalance() < value) throw new Exception("Not enough money!");
+
+   long balance = car.getWallet().getBalance();
+
+    long chargerBalance = wallet.getBalance();
+
+
+    balance = balance -value;
```
Comment (AndreyBronin) [inline backend/src/main/java/com/v2gdemo/backend/service/BillingService.java]: balance -= value;
    chargerBalance += value;

File: backend/src/main/java/com/v2gdemo/backend/service/BillingService.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    chargerBalance = chargerBalance+value;
+
+
+    car.getWallet().setBalance(balance);
+    wallet.setBalance(chargerBalance);
+
```
Comment (AndreyBronin) [inline backend/src/main/java/com/v2gdemo/backend/service/BillingService.java]: create new Transaction here!

File: backend/src/main/java/com/v2gdemo/backend/service/BillingService.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+@Service
+public class BillingService {
+  private WalletRepository walletRepository;
+  private ObjectRepository objectRepository;
+  public BillingService(WalletRepository walletRepository, ObjectRepository objectRepository){
+    this.objectRepository = objectRepository;
+    this.walletRepository = walletRepository;
+  }
+  public synchronized void makeTransaction(Object car, String address, long value) throws Exception{
```
Comment (AndreyBronin) [inline backend/src/main/java/com/v2gdemo/backend/service/BillingService.java]: makeTransaction(String fromWallet, String toWallet, long value)

Comment (AndreyBronin) [reply backend/src/main/java/com/v2gdemo/backend/service/BillingService.java]: also API method in controller needed for makeTransaction

Comment (NothingIsMatter) [reply backend/src/main/java/com/v2gdemo/backend/service/BillingService.java]: K, will be done!

**Pull request merged**